web/src/components/organisms/Settings/Integration/index.tsx

```diff
@@ -0,0 +1,128 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import ConfigProvider from "@reearth-cms/components/atoms/ConfigProvider";
+import Content from "@reearth-cms/components/atoms/Content";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import PageHeader from "@reearth-cms/components/atoms/PageHeader";
+import Search from "@reearth-cms/components/atoms/Search";
+import Table from "@reearth-cms/components/atoms/Table";
```

Comment:
I already said this to @nourbalaha, apologies if I haven't noticed this up until now and have let quite a bit into the source code, but atoms shouldn't be used in an organism.

Pages are built up from organisms, organisms are built up from molecules, molecules are built up from other molecules and atoms, and atoms are built up from other atoms.